source/core/coverage/ut_coverage.pkb

```diff
@@ -84,8 +85,9 @@ create or replace package body ut_coverage is
                where s.owner = el.owner and  s.name = el.name
            )
        and line > 0
+       {regex_filters}
```

Comment:
Why was this added as a separate filter at the end instead of being added to the main filter in top WITH clause?

Reply:
Good point. I have to review some things after your recent PR to improve COVECOV sql anyway.